src/components/Reports/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import './index.scss';
+function reports() {
+  return (
+    <div className="reports">
+      soy los reports <span className="reports__wi">WIIIIIIIIIIIIIIIIIII</span>!!!!!!!!
+    </div>
+  );
+}
```

Comment:
Intentemos usar arrow functions en vez de `function`. De esta manera se facilita el código y queda mas conciso.
```suggestion
const reports = () => (
  <div className="reports">
    soy los reports <span className="reports__wi">WIIIIIIIIIIIIIIIIIII</span>!!!!!!!!
  </div>
);
```

Observar que en este bloque no hace falta hacer un `return` ya que es implicito el resultado aquí.